### PR TITLE
[ Send ] character creation update

### PR DESCRIPTION
**Pull request by alisonrag**
add new config Key
```
# Character Creation Version
# 1 = Classic Version, 2 = Classic Version without attribute, 3 = Two Sex Account Version , 4 = Doram Version
charCreateVersion 1
```

tested in bRO ( this server dont have any pre-configured version, still use old version 0067)
Dialog:
bro dialog

Confirmation:
bro confimation

tested in iRO (this server have create_version = 0x0A39 in servertype)
Dialog:
iro dialog

Confirmation:
iro confirmation

this pull have compatibility with old creation functions

### CODE REVIEW

File: control/config.txt [hunk 1]
```diff
@@ -10,6 +10,10 @@ password
 loginPinCode
 char
 
+# Character Creation Version
+# 1 = Classic Version, 2 = Classic Version without attribute, 3 = Two Sex Account Version , 4 = Doram Version
+charCreateVersion 1
```
Comment (sctnightcore) [inline control/config.txt]: My idea : add on server.txt

Comment (alisonrag) [reply control/config.txt]: @allanon  @lututui  what do you think about?

Comment (alisonrag) [reply control/config.txt]: @sctnightcore 
if averyone agree i can move this to servers.txt

**Comment by alisonrag** [comment]
moving to servers.txt and documenting

File: tables/servers.txt [hunk 2]
```diff
@@ -113,7 +118,8 @@ paddedPackets 0
 paddedPackets_attackID 0x72
 paddedPackets_skillUseID 0xF5
 addTableFolders iRO
-charBlockSize 145
+charBlockSize 145]
```
Comment (sctnightcore) [inline tables/servers.txt]: :p

**Comment by alisonrag** [comment]
i will wait for bRO and iRO update and i will re-pull

**Pull request closed without merging**